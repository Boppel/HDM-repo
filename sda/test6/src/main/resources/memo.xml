<?xml version="1.0" encoding="UTF-8"?>
<memo>
  <from>M. Goik</from>
  <to>B. King</to>
  <to>A. June</to>
  <subject>Best wishes</subject>
  <content>Hi all, congratulations to your splendid party</content>
</memo>
