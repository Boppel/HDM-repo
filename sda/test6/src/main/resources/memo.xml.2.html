<html>
<head><title>Memo from M. Goik</title></head>
<body>
<h2>Subject:Best wishes</h2>
<dl>
<dt>Sender:</dt>
<dd>M. Goik</dd>
<dt>Recipients:</dt>
<dd>
<ul>
<li>B. King</li>
<li>A. June</li>
</ul>
</dd>
</dl>
<h2>SubjectBest wishes</h2>
<p>Hi all, congratulations to your splendid party</p>
<p>End of Message from <strong>M. Goik</strong></p>
</body>
</html>
