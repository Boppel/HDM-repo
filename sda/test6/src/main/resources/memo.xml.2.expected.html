<html>
  <head><title>Memo from M. Goik</title></head>
  <body>
    <h2>Subject:Best wishes</h2>
    <dl>
      <dt>Sender:</dt>
      <dd>M. Goik</dd>
      <dt>Recipients:</dt>
      <dd>
        <ul>
          <li>B. King</li>
          <li>A. June</li>
        </ul>
      </dd>
    </dl>
    <h2>Subject: Best wishes</h2>
    <p>Hi all, congratulations to your splendid party</p>
    <p>End of message from <strong>M. Goik</strong></p>
  </body>
</html>
